packages/components/src/tooltip/index.js

```diff
@@ -87,7 +91,14 @@ const emitToChild = ( children, eventName, event ) => {
 	}
 };
 
-function Tooltip( { children, position, text, shortcut } ) {
+function Tooltip( {
+	children,
+	className,
+	position,
+	text,
+	shortcut,
+	...popoverProps
```

Comment:
Hmmmm 🤔 This seems to be a rather big modification. I wonder if we should use an explicit `popoverProps` property instead of `...` rest. We need to update the README too.

As a last thought, maybe we should move this to a separate PR?